src/test/java/edu/kis/powp/jobs2d/ExceededCanvasTest.java

```diff
@@ -0,0 +1,76 @@
+package edu.kis.powp.jobs2d;
+
+
+import edu.kis.powp.jobs2d.command.CompoundCommand;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.visitor.ExceededCanvasVisitor;
+import org.junit.Test;
+
+import java.awt.*;
+import java.awt.geom.Rectangle2D;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class ExceededCanvasTest {
```

Comment:
OK, but please, add also an option in the menu to test the current command against some standard canvases.